class.jetpack-network.php

```diff
@@ -477,6 +477,16 @@ public function do_subsiteregister( $site_id = null ) {
 			'timeout' => $timeout,
 		);
 
+		list( $activation_source_name, $activation_source_keyword ) = get_option( 'jetpack_activation_source', false );
```

Comment:
This will be different in php5 vs. php7, I believe. http://php.net/manual/en/function.list.php

Reply:
How so, exactly?

Reply:
NM, I misunderstood